napari/components/viewer_model.py

```diff
@@ -383,17 +383,100 @@ def _sliced_extent_world_augmented(self) -> np.ndarray:
         return self.layers._extent_world_augmented[:, self.dims.displayed]
 
     def reset_view(
-        self, *, margin: float = 0.05, reset_camera_angle: bool = True
+        self, *, margin: float = 0.05, reset_camera_angle: bool | None = None
     ) -> None:
-        """Reset the camera view.
+        """Reset the camera and fit the current layers to the canvas.
+
+        Resets the angles of the camera, adjust the camera zoom,
+        and centers the view so that all layers are visible,
+        accounting for the current grid mode and margin.
 
         Parameters
         ----------
         margin : float in [0, 1)
             Margin as fraction of the canvas, showing blank space around the
-            data.
+            data. Default is 0.05 (5% of the canvas).
+        reset_camera_angle : bool
+            Whether to reset the camera angles to (0, 0, 90) before fitting
+            to view. Default is True.
+            .. deprecated:: 0.6.0
```

Comment:
```suggestion
            to view. Default is True.

            .. deprecated:: 0.6.0
```

I'm not sure but I think sphinx directives need a blank newline in front of them.

Reply:
I was going off based of TextManager deprecation format, but that is not passed to docs API build. Instead, I found this formatting which does build in docs correctly:
https://github.com/napari/napari/blob/c36a49d61c348a1c24166391421482da1e3aa2e3/napari/_qt/qt_main_window.py#L1171-L1176